src/latis-python-client/latis.py

```diff
@@ -78,9 +150,17 @@ def __getCatalog(self):
 
 
 class Catalog:
+    """Catalog for a Latis instance.
 
-    def __init__(self, latisInstance):
+    Attributes:
+        datasets (dict): Dictionary: {Formatted name: latis name}.
+        list (numpy.ndarray): List of all dataset latis names in catalog.
 
+    Args:
+        latisInstance (LatisInstance): The Latis instance.
```

Comment:
It suggests: `latisInstance (LatisInstance): The Latis instance this catalog is associated with.`